wallet/src/cli.rs

```diff
@@ -21,9 +21,9 @@ pub struct Cli {
     /// RPC endpoint of the node that this wallet will connect to.
     pub endpoint: String,
 
-    #[arg(long, short)]
+    #[arg(long, short('d'))]
     /// Path where the wallet data is stored. Default value is platform specific.
-    pub path: Option<PathBuf>,
+    pub base_path: Option<PathBuf>,
```

Comment:
The wallet now uses the same long and short flag as the node itself, and this frees up `-p` for the new `--parachain` flag.